WebContent/js/controllers/FilterController.js

```diff
@@ -0,0 +1,39 @@
+mobileStoreApp.controller('FilterController', ['$scope', function($scope){
+	
+	/* show initial filter options  */
+    $scope.showFilters = function(){
+        $(".slidingDiv_brands").hide();
```

Comment:
Please check the alternative for DOM manipulation in Angular so that we can get rid of JQuery.